packages/shared/src/lib/image.ts

```diff
@@ -58,8 +58,12 @@ export const cloudinary = {
     },
     imageFallback:
       'https://daily-now-res.cloudinary.com/image/upload/s--LrHsyt2T--/f_auto/v1692632054/squad_placeholder_sfwkmj',
-    createSquad:
-      'https://daily-now-res.cloudinary.com/image/upload/s--4nZR_i2V--/f_auto/v1686299591/Squads_eary_access_e3lo4u',
+    createSquad: {
+      beyondMobile:
```

Comment:
What's beyond mobile? :jenkins-triggered:

Maybe just keep `mobile/desktop` naming think it's clear enough

Reply:
Desktop is a bit inaccurate since it can also be applied to tablets.